src/plugins/vis_augmenter/public/view_events_flyout/components/styles.scss

```diff
@@ -0,0 +1,63 @@
+$vis-description-width: 200px;
+$event-vis-height: 55px;
+$timeline-panel-height: 100px;
+$content-padding-top: 110px; // Padding needed within view events flyout content to sit comfortably below flyout header
+$date-range-height: 45px; // Static height we want for the date range picker component
+$error-icon-padding-right: -8px; // This is so the error icon is aligned consistent with the event count icons
+$base-vis-min-height: 25vh; // Visualizations require the container to have a valid width and height to render
+
+.view-events-flyout {
+  &__baseVis {
+    min-height: $base-vis-min-height;
+  }
+
+  &__eventVis {
+    height: $event-vis-height;
+  }
+
+  &__timelinePanel {
+    height: $timeline-panel-height;
+  }
+
+  &__visDescription {
+    min-width: $vis-description-width;
+    max-width: $vis-description-width;
+    word-break: break-word;
+  }
+
+  &__content {
+    position: absolute;
+    top: $content-padding-top;
+    right: $euiSizeM;
+    bottom: $euiSizeM;
+    left: $euiSizeM;
+  }
+
+  &__contentPanel {
+    @include euiYScroll;
+
+    overflow: auto;
+    overflow-x: hidden;
+    scrollbar-gutter: stable both-edges;
+  }
+}
+
+.hide-y-scroll {
```

Comment:
I feel like we want to add this off the `.view-events-flyout` still since I can't be for certain that this isn't be referenced any where else.

Reply:
Sure - will move into there.